setup.py

```diff
@@ -26,14 +26,14 @@ def package_data(pkg, roots):
 
 setup(
     name="openedx-scorm-xblock",
-    version="16.0.2",
+    version="16.0.3",
     description="Scorm XBlock for Open edX",
     long_description=readme,
     long_description_content_type="text/x-rst",
     author="Overhang.IO",
     author_email="contact@overhang.io",
     maintainer="Overhang.IO",
-    maintainer_email="regis@overhang.io",
+    maintainer_email="regis@overhang.io, zia.fazal@edly.io",
```

Comment:
Since you are taking over as maintainer, you can safely remove my email address.